dev/docker-compose.yaml

```diff
@@ -276,6 +276,24 @@ services:
     depends_on:
       - redis
 
+  isaccml:
+    image: ghcr.io/uwcirg/isacc-ml:${ISACCML_IMAGE_TAG:-main}
+    environment:
+      PREFERRED_URL_SCHEME: https
+      LOGSERVER_URL: "https://logs.${BASE_DOMAIN:-localtest.me}"
+    env_file:
+      - isaccml.env
+    volumes:
+      - ./config/isaccml:/opt/app/config:ro
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.isaccml-${COMPOSE_PROJECT_NAME}.rule=Host(`isaccml.${BASE_DOMAIN:-localtest.me}`)"
+      - "traefik.http.routers.isaccml-${COMPOSE_PROJECT_NAME}.entrypoints=websecure"
+      - "traefik.http.routers.isaccml-${COMPOSE_PROJECT_NAME}.tls=true"
+      - "traefik.http.routers.isaccml-${COMPOSE_PROJECT_NAME}.tls.certresolver=letsencrypt"
```

Comment:
if I understand correctly, this service will (at least initially) only be called by other isacc services, and does not need to internet-accessible (eg to the enduser)

if that's the case, this traefik config will not be needed
```suggestion
```